appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/util/ResourceValidator.java

```diff
@@ -1036,6 +1037,18 @@ public boolean find(String jndiName, JndiNameEnvironment env) {
                 return nonPortableJndiNames.contains(jndiName);
             }
         }
+       
+        /**
+         * Remove suffix from the module name.
+         */
+        private String getActualModuleName(String moduleName) {
+            if(moduleName != null){
```

Comment:
Tiny small thing, but convention for formatting asks for some extra spaces here:
```java
if (moduleName != null) {
```